make.js

```diff
@@ -262,6 +271,9 @@ target.test = function() {
         fail(`Unable to find tests using the following patterns: ${JSON.stringify([pattern1, pattern2, pattern3])}`);
     }
 
+    // install and use node 6 to run the tests
+    util.installNode('v6.10.3', buildTestsPath);
```

Comment:
```js
if (process.env.TF_BUILD) {
  ...
}
```

Reply:
Another way that would be more clean would be to expose a command line option whether to use node 5 for testing, otherwise default to node 6. That way the CI is running the same code that runs on users machines.

Does this download node every time? I have some code somewhere that will cache it under $home